python/astro_metadata_translator/translators/fits.py

```diff
@@ -38,7 +39,9 @@ class FitsTranslator(MetadataTranslator):
 
     # Direct translation from header key to standard form
     _trivial_map: Dict[str, Union[str, List[str], Tuple[Any, ...]]] = dict(
-        instrument="INSTRUME", telescope="TELESCOP"
+        instrument="INSTRUME",
+        telescope="TELESCOP",
+        focus_z=("FOCUSZ", dict(unit=u.mm, default=0.0, checker=no_defocal_info)),
```

Comment:
We should not put the default implementation here because `FOCUSZ` doesn't seem to be part of the FITS standard. The default implementation should go in `MetadataTranslator`.